installers/android-sdk/installer.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
+
+YUM_CMD=$(which yum) # yum package manager for RHEL & CentOS
+DNF_CMD=$(which dnf) # dnf package manager for new RHEL & CentOS
+APT_GET_CMD=$(which apt-get) # apt package manager for Ubuntu & other Debian based distributions
+PACMAN_CMD=$(which pacman) # pacman package manager for ArchLinux
+APK_CMD=$(which apk) # apk package manager for Alpine
+
+ if [ ! -z $APT_GET_CMD ]; then
+    sudo apt-get update
+    sudo apt-get install android-sdk
+ elif [ ! -z $APK_CMD ]; then
+    sudo apk add --no-cache bash unzip libstdc++
+    mkdir -p /opt/android-sdk && cd /opt/android-sdk
+    && wget -q http://dl.google.com/android/repository/tools_r27.0.0-linux.zip -O android-sdk-tools.zip
+    && unzip -q android-sdk-tools.zip -d /opt/android-sdk
+    && rm -f android-sdk-tools.zip
+    && chmod 777 /opt/android-sdk/*
```

Comment:
setting to `777` is dangerous, right?

Reply:
Oh, yes sure. I just changed it to `755`.